src/main/resources/data/shardthief/games/cyber_block.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
This doesn't have consistency with the other game IDs or this game's name or structure ID.

Reply:
Change it then.